### ISSUE TITLE
v6.0.0-alpha.1 for vue - JSX loader seems missing after fresh install 

### ISSUE THREAD

**Issue by silvio-e** [storybookjs/storybook#9637]
**Describe the bug**
After a fresh install of v6.0.0-alpha.1 the jsx version of the button example would cause an error

**To Reproduce**
1. Install with npx -p @storybook/cli@next sb init --type vue
2. Start with yarn storybook
3. See error in Browser

**Expected behavior**
Shows all example buttons correctly

**Screenshots**

<img width="1039" alt="Bildschirmfoto 2020-01-26 um 15 12 03" src="https://user-images.githubusercontent.com/3109921/73137271-4bb4fa80-404e-11ea-87d7-88c2d71402a2.png">

**Comment by cdll** [comment]
same issue here～

**Comment by shilman** [comment]
cc @Aaron-Pool can you help with this?

**Comment by Aaron-Pool** [comment]
@cdll mind duplicating your setup and pushing it to a public repo?

**Comment by cdll** [comment]
@Aaron-Pool oh, hi, sorry to see your reply late, here 's my reproduce repo:
https://github.com/cdll/storybook-v6alpha-vue-jsxLoader-issue-repro

**Comment by Aaron-Pool** [comment]
@cdll your setup is missing the `@vue/app` Babel preset. You need to install it and add it to your `babel.config.js`.

@shilman that's a step the user is supposed to setup manually, correct? Unless he's using the cli, I suppose?

**Comment by cdll** [comment]
@Aaron-Pool thank you for that after installed ``@vue/babel-preset-app`` and ``babel-preset-env`` my demo had been run up but with warning such as below: 
````shell
> start-storybook -p 6006

info @storybook/vue v6.0.0-alpha.25
info 
info => Loading presets
info => Loading presets
info => Loading config/preview file in "./.storybook".
info => Adding stories defined in ".storybook/main.js".
info => Using default Webpack setup.
webpack built da4537f3b22af68a4261 in 11145ms
⚠ ｢wdm｣: Hash: da4537f3b22af68a4261
Version: webpack 4.42.0
Time: 11145ms
Built at: 2020-03-19 11:43:09 PM
                                          Asset      Size        Chunks                                Chunk Names
                                    iframe.html  2.87 KiB                [emitted]                     
            main.da4537f3b22af68a4261.bundle.js  10.1 KiB          main  [emitted] [immutable]         main
        main.da4537f3b22af68a4261.bundle.js.map  1.65 KiB          main  [emitted] [dev]               main
    runtime~main.da4537f3b22af68a4261.bundle.js  31.2 KiB  runtime~main  [emitted] [immutable]         runtime~main
runtime~main.da4537f3b22af68a4261.bundle.js.map  32.3 KiB  runtime~main  [emitted] [dev]               runtime~main
    vendors~main.da4537f3b22af68a4261.bundle.js  2.53 MiB  vendors~main  [emitted] [immutable]  [big]  vendors~main
vendors~main.da4537f3b22af68a4261.bundle.js.map  2.64 MiB  vendors~main  [emitted] [dev]               vendors~main
Entrypoint main [big] = runtime~main.da4537f3b22af68a4261.bundle.js runtime~main.da4537f3b22af68a4261.bundle.js.map vendors~main.da4537f3b22af68a4261.bundle.js vendors~main.da4537f3b22af68a4261.bundle.js.map main.da4537f3b22af68a4261.bundle.js main.da4537f3b22af68a4261.bundle.js.map
[0] multi ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.storybook/storybook-init-framework-entry.js ./node_modules/@storybook/addon-links/dist/preset/addDecorator.js-generated-config-entry.js ./.storybook/generated-stories-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false 88 bytes {main} [built]
[./.storybook/generated-stories-entry.js] 230 bytes {main} [built]
[./.storybook/storybook-init-framework-entry.js] 24 bytes {main} [built]
[./node_modules/@storybook/addon-links/dist/preset/addDecorator.js] 214 bytes {vendors~main} [built]
[./node_modules/@storybook/addon-links/dist/preset/addDecorator.js-generated-config-entry.js] 513 bytes {vendors~main} [built]
[./node_modules/@storybook/client-api/dist/index.js] 4.46 KiB {vendors~main} [built]
[./node_modules/@storybook/core/dist/server/common/polyfills.js] 120 bytes {vendors~main} [built]
[./node_modules/@storybook/core/dist/server/preview/globals.js] 93 bytes {vendors~main} [built]
[./node_modules/@storybook/vue/dist/client/index.js] 1.21 KiB {vendors~main} [built]
[./node_modules/airbnb-js-shims/index.js] 40 bytes {vendors~main} [built]
[./node_modules/core-js/features/symbol/index.js] 359 bytes {vendors~main} [built]
[./node_modules/global/window.js] 232 bytes {vendors~main} [built]
[./node_modules/querystring-es3/index.js] 127 bytes {vendors~main} [built]
[./node_modules/regenerator-runtime/runtime.js] 23.6 KiB {vendors~main} [built]
[./node_modules/webpack-hot-middleware/client.js?reload=true&quiet=false] (webpack)-hot-middleware/client.js?reload=true&quiet=false 7.68 KiB {vendors~main} [built]
    + 586 hidden modules

WARNING in ./stories/0-Welcome.stories.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
TypeError: /Users/cdll/github/sb/stories/0-Welcome.stories.js: Cannot read property 'bindings' of null
    at Scope.moveBindingTo (/Users/cdll/github/sb/node_modules/@babel/traverse/lib/scope/index.js:830:13)
    at convertBlockScopedToVar (/Users/cdll/github/sb/node_modules/babel-plugin-transform-es2015-block-scoping/lib/index.js:139:13)
    at PluginPass.VariableDeclaration (/Users/cdll/github/sb/node_modules/babel-plugin-transform-es2015-block-scoping/lib/index.js:26:9)
    at newFn (/Users/cdll/github/sb/node_modules/@babel/traverse/lib/visitors.js:179:21)
    at NodePath._call (/Users/cdll/github/sb/node_modules/@babel/traverse/lib/path/context.js:55:20)
    at NodePath.call (/Users/cdll/github/sb/node_modules/@babel/traverse/lib/path/context.js:42:17)
    at NodePath.visit (/Users/cdll/github/sb/node_modules/@babel/traverse/lib/path/context.js:90:31)
    at TraversalContext.visitQueue (/Users/cdll/github/sb/node_modules/@babel/traverse/lib/context.js:112:16)
    at TraversalContext.visitSingle (/Users/cdll/github/sb/node_modules/@babel/traverse/lib/context.js:84:19)
    at TraversalContext.visit (/Users/cdll/github/sb/node_modules/@babel/traverse/lib/context.js:140:19)
    at Function.traverse.node (/Users/cdll/github/sb/node_modules/@babel/traverse/lib/index.js:84:17)
    at NodePath.visit (/Users/cdll/github/sb/node_modules/@babel/traverse/lib/path/context.js:97:18)
    at TraversalContext.visitQueue (/Users/cdll/github/sb/node_modules/@babel/traverse/lib/context.js:112:16)
    at TraversalContext.visitMultiple (/Users/cdll/github/sb/node_modules/@babel/traverse/lib/context.js:79:17)
    at TraversalContext.visit (/Users/cdll/github/sb/node_modules/@babel/traverse/lib/context.js:138:19)
    at Function.traverse.node (/Users/cdll/github/sb/node_modules/@babel/traverse/lib/index.js:84:17)
 @ \.)(?=.)[^\/]*?\.stories\.js\/?)$ (./stories sync ^\.\/(?:(?:(?!\.)(?:(?:(?!(?:|\/)\.).)*?)\/)?(?!\.)(?=.)[^\/]*?\.stories\.js\/?)$) ./0-Welcome.stories.js
 @ ./.storybook/generated-stories-entry.js
 @ multi ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.storybook/storybook-init-framework-entry.js ./node_modules/@storybook/addon-links/dist/preset/addDecorator.js-generated-config-entry.js ./.storybook/generated-stories-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false

WARNING in ./stories/1-Button.stories.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
TypeError: /Users/cdll/github/sb/stories/1-Button.stories.js: Cannot read property 'bindings' of null
    at Scope.moveBindingTo (/Users/cdll/github/sb/node_modules/@babel/traverse/lib/scope/index.js:830:13)
    at convertBlockScopedToVar (/Users/cdll/github/sb/node_modules/babel-plugin-transform-es2015-block-scoping/lib/index.js:139:13)
    at PluginPass.VariableDeclaration (/Users/cdll/github/sb/node_modules/babel-plugin-transform-es2015-block-scoping/lib/index.js:26:9)
    at newFn (/Users/cdll/github/sb/node_modules/@babel/traverse/lib/visitors.js:179:21)
    at NodePath._call (/Users/cdll/github/sb/node_modules/@babel/traverse/lib/path/context.js:55:20)
    at NodePath.call (/Users/cdll/github/sb/node_modules/@babel/traverse/lib/path/context.js:42:17)
    at NodePath.visit (/Users/cdll/github/sb/node_modules/@babel/traverse/lib/path/context.js:90:31)
    at TraversalContext.visitQueue (/Users/cdll/github/sb/node_modules/@babel/traverse/lib/context.js:112:16)
    at TraversalContext.visitSingle (/Users/cdll/github/sb/node_modules/@babel/traverse/lib/context.js:84:19)
    at TraversalContext.visit (/Users/cdll/github/sb/node_modules/@babel/traverse/lib/context.js:140:19)
    at Function.traverse.node (/Users/cdll/github/sb/node_modules/@babel/traverse/lib/index.js:84:17)
    at NodePath.visit (/Users/cdll/github/sb/node_modules/@babel/traverse/lib/path/context.js:97:18)
    at TraversalContext.visitQueue (/Users/cdll/github/sb/node_modules/@babel/traverse/lib/context.js:112:16)
    at TraversalContext.visitMultiple (/Users/cdll/github/sb/node_modules/@babel/traverse/lib/context.js:79:17)
    at TraversalContext.visit (/Users/cdll/github/sb/node_modules/@babel/traverse/lib/context.js:138:19)
    at Function.traverse.node (/Users/cdll/github/sb/node_modules/@babel/traverse/lib/index.js:84:17)
 @ \.)(?=.)[^\/]*?\.stories\.js\/?)$ (./stories sync ^\.\/(?:(?:(?!\.)(?:(?:(?!(?:|\/)\.).)*?)\/)?(?!\.)(?=.)[^\/]*?\.stories\.js\/?)$) ./1-Button.stories.js
 @ ./.storybook/generated-stories-entry.js
 @ multi ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.storybook/storybook-init-framework-entry.js ./node_modules/@storybook/addon-links/dist/preset/addDecorator.js-generated-config-entry.js ./.storybook/generated-stories-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false
Child HtmlWebpackCompiler:
                          Asset      Size               Chunks  Chunk Names
    __child-HtmlWebpackPlugin_0  6.32 KiB  HtmlWebpackPlugin_0  HtmlWebpackPlugin_0
    Entrypoint HtmlWebpackPlugin_0 = __child-HtmlWebpackPlugin_0
    [./node_modules/html-webpack-plugin/lib/loader.js!./node_modules/@storybook/core/dist/server/templates/index.ejs] 2.11 KiB {HtmlWebpackPlugin_0} [built]
╭──────────────────────────────────────────────────────╮
│                                                      │
│   Storybook 6.0.0-alpha.25 started                   │
│   11 s for manager and 12 s for preview              │
│                                                      │
│    Local:            http://localhost:6006/          │
│    On your network:  http://192.168.199.124:6006/    │
│                                                      │
╰──────────────────────────────────────────────────────╯
````
what i really want you to know is that my demo repo is created by storybook-cli even with the packages deps and devDeps. 
could also be reproduce by that repo https://github.com/cdll/storybook-v6alpha-vue-jsxLoader-issue-repro. thanks for more help~

**Comment by Aaron-Pool** [comment]
@cdll right, unfortunately I don't have any knowledge of the storybook/cli codebase, but you're right that it does seem like it shouldn't be throwing warnings or errors out of the box. I'll try to look into that.

**Comment by cdll** [comment]
@Aaron-Pool so much thanks. hopes for good news soon~

**Comment by shilman** [comment]
@Aaron-Pool the CLI simply copies from a template directory. It's very possible that the template is wrong. It should be a straighforward fix -- feel free to DM if you have any q's

**Comment by cdll** [comment]
hi all, how 's going~
the issue 's still even after i 've upgraded to ``"@storybook/vue": "^6.0.0-alpha.29"``

**Comment by graup** [comment]
Hi all, could you try running the latest beta version? We've now removed the JSX example stories from the generator. If you need JSX, check the migration guide for a manual setup.

**Comment by may17** [comment]
Hi all, seems that the problem still exists. I`ve installed version 6.1.11. Didn`t find anything about jsx in the guide? Or did i miss something here https://storybook.js.org/blog/storybook-6-migration-guide/ ?

**Comment by may17** [comment]
ok seems like this is the real migration guide with the topic jsx topic included https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#removed-babel-preset-vue-from-vue-preset

**Comment by vanessayuenn** [comment]
Hi! I am going through and cleaning up old issues in the repo. We just shipped a ton of improvements in Storybook 7.0 (especially for vue!), and I suspect this bug might be fixed already. Can you please try it out? If this issue persists with the latest version, please open a new bug report with a 7.x reproduction. Thank you so much! 🙏🏼
